keyboards/naked60/keymaps/default/readme.md

```diff
@@ -0,0 +1,356 @@
+# The default keymap for naked60
+
+Default<br>
+<table>
```

Comment:
Please use GFM syntax, it's much easier to read: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables